Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Rapid7_Enrichment.yml

```diff
@@ -0,0 +1,1004 @@
+                root: Nexpose
+                accessor: AssetTag
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 750,
+          "y": 570
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: b971196e-d62d-446c-8b48-dabe3ed2d9ea
+    type: condition
+    task:
+      id: b971196e-d62d-446c-8b48-dabe3ed2d9ea
+      version: -1
+      name: Was there an owner asset tag from Rapid7?
+      description: Checks if the Nexpose AssetTag object exists with an "owner" tag.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "19"
+      "yes":
+      - "7"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: Nexpose.AssetTag
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: Nexpose.AssetTag.Type
+                      iscontext: true
+                    right:
+                      value:
+                        simple: owner
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1190,
+          "y": 570
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "23":
+    id: "23"
+    taskid: 0ee49a95-239a-4839-823d-ff066c9fe6a0
+    type: regular
+    task:
+      id: 0ee49a95-239a-4839-823d-ff066c9fe6a0
+      version: -1
+      name: Set tags grid field
+      description: |-
+        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. For example:
+        `!GridFieldSetup keys=ip,src val1=${AWS.EC2.Instances.NetworkInterfaces.PrivateIpAddress} val2="AWS" gridfiled="gridfield"`
+      scriptName: GridFieldSetup
+      type: regular
+      iscommand: false
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "19"
+    scriptarguments:
+      gridfield:
+        simple: asmtags
+      keys:
+        simple: Key,Value,Source
+      val1:
+        complex:
+          root: Nexpose.AssetTag
+          filters:
+          - - operator: isNotEqualString
+              left:
+                value:
+                  simple: Nexpose.AssetTag.Type
+                iscontext: true
+              right:
+                value:
+                  simple: owner
```

Comment:
Nitty question: Is it possible that only the owner `AssetTag.Type` could be found and therefore this would fail?  The previous conditional seems to only check for `Nexpose.AssetTag` and not `Nexpose.AssetTag where type != owner` so I think there might be a potential failure scenario though I don't know how rare it is.  Will leave it up to you to change if you please.

Reply:
<img width="974" alt="image" src="https://user-images.githubusercontent.com/40349459/220978606-4990289c-fe6a-43d2-9095-1a38e3872c82.png">

Resolved in b2ffd19.